addons/alerting.libsonnet

```diff
@@ -220,64 +132,19 @@ function(config) {
           equal:
           - alertname
         receivers:
-        - name: Black_Hole
```

Comment:
We get waaaaaaaaay too many alerts from upstream by default, and some of them really shouldn't go anywhere (e.g. [infoinhibitor](https://runbooks.prometheus-operator.dev/runbooks/general/infoinhibitor/)). That is why the black_hole is our default receiver 🙂 

Making too many changes to one PR can be quite harmful, because if this change on the default receiver brings some unexpected behavior, then we'll need to revert all changes here. Including the per-team slack channels 😬 

We can consider changing the default receiver in future iterations!